packages/liveblocks-react/.eslintrc.js

```diff
@@ -24,6 +25,12 @@ module.exports = {
     "@typescript-eslint/no-inferrable-types": "off",
     "no-constant-condition": "off",
 
+    // -------------------------------
+    // Enforce React best practices
+    // -------------------------------
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "error",
```

Comment:
These rules are essential in every React project that uses hooks and should always be enforced/respected.